<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salesmen</class>
 <widget class="QDialog" name="Salesmen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кассиры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QTableWidget" name="TW_salesmen">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Процент комисионных</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комиссионные</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="LE_name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_fees">
       <property name="placeholderText">
        <string>Комиссионные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_line">
       <property name="placeholderText">
        <string>№ строки (для увольнения)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_hire">
       <property name="text">
        <string>Нанять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_fire">
       <property name="text">
        <string>Уволить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
